### ISSUE TITLE
Make 'brady bunch' layout usable.

### ISSUE THREAD

**Issue by guusdk** [igniterealtime/openfire-ofmeet-plugin#91]
Jitsi Meet recently added a tile view mode (also referred to as "the Brady Bunch" mode), which allows the video for all participants to be shown at once (in tiles of equal side).

Although the code for this is already available in OFMeet 0.9.5, it currently cannot be configured. The tileview toolbar button should be added to the available toolbar buttons.

**Comment by lmagyar** [comment]
Yes, this would be very useful eg. in a teacher/student relation, where the teacher wants to see the students' streams all at once!

**Comment by niente0** [comment]
Any update? Is it possible to activate it manually, for example by adding a new property in Openfire? Thanks

**Comment by niente0** [comment]
Hello, I figured out by myself. For anyone who needs to activate this feature, just add this system property:
**ofmeet.buttons.enabled.00019** -> **tileview**
Bye!!
